README.md

```diff
@@ -159,93 +122,69 @@ To use this convenience file, create a text file with the following format:
     password=changeme
     # Access scheme (default: https)
     scheme=https
-    # Your version of Splunk (default: 5.0)
-    version=5.0
+    # Your version of Splunk
+    version=8.0
```

Comment:
Your update is correct, but our tests are still running against 6.6 and 7.0.

Reply:
I changed it to 7.0